docs/getting_started/preparation.md

```diff
@@ -1,23 +1,147 @@
+        [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
+        [--config-env=<name>=<envvar>] <command> [<args>]
+```
+**TODO** Show results from an actual power shell
 
-    - download
-    - install
+### Install on Linux
+
+Git is shipped with the package manager of all common distributions and you can install it from the command line.
+See [here](https://git-scm.com/download/linux) to get the command for your distribution, e.g.:
+
+```bash
+sudo apt-get install git  # On Debian/Ubuntu
+sudo pacman -S git  # On Arch linux
+```
 
 ## Python
 
-!!! todo
+Flowcean is written in Python and requires at least version 3.12.
+To check the version of your Python installation, open a Powershell or command line and type
+
+```bash
+> python3 --version
+Python 3.12.4
+```
+
+If the result looks like `Python 3.12.x` (where `x` might be any number), then you're settled and you can continue to the next section.
+
+### Install on Windows
+
+Nowadays, Windows offers to options for development.
+First, you can use plain Windows, e.g., using the Powershell.
+Second, there is the Windows Subsystem for Linux (WSL), which uses an integrated virtual machine for linux.
+Although we would recommend going the second route, this might not be applicable for everyone.
+
+#### Python on Powershell
+
+First, you need to download the installer for Python. 
+You will always find the latest version on [https://www.python.org/downloads/windows/](https://www.python.org/downloads/windows/).
+Once downloaded, double-click on the installer and follow the instructions; most, if not all, settings can be kept at their defaults.
+
+As additional step, to be able to use virtual environments, you have to open a powershell in administrator mode and type
+
+```PS
+Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
+```
+
+Afterwards, it may be necessary to restart your computer.
+
+Finally, you should be able to successfully check the installation as described above. 
+
+#### Python on WSL
+
+Open a powershell in administrator mode and run the command
+
+```PS
+wsl --install
+```
 
-    - install
-    - `python --version`
+which will install WSL and Ubuntu on your machine.
+Afterwards, you have to restart your computer and then follow the instructions at [Install on Linux](#install-on-linux-1)
+
+If you have an older version of Windows or prefer a step-by-step guide, please refer to [https://learn.microsoft.com/en-us/windows/wsl/install-manual](https://learn.microsoft.com/en-us/windows/wsl/install-manual)
+
+
+
+### Install on Linux
+
+Most linux distributions ship Python by default. 
+However, the version might be different from what is needed for Flowcean (>= 3.12).
+If you're on Debian/Ubuntu, you have to execute a sequence of commands:
+
+```bash
+sudo apt update && sudo apt upgrade -y
+sudo apt install software-properties-common -y
+sudo add-apt-repository ppa:deadsnakes/ppa
+sudo apt update && sudo apt install python3.12
+```
```

Comment:
Also here: we might just need any python or even better just `hatch`

Reply:
But in the pyproject file the version (>=3.12) is pinned, i.e., installation in a virtual environment will fail. (Regarding hatch, see my comment above)

Reply:
If you want to manually manage your python installations, this is a tedious task of finding out which python version works etc. Same as above, mentioning the versions in the documentation forces us to keep them in sync with the metadata in the code...